doc/rados/operations/health-checks.rst

```diff
@@ -1678,17 +1678,22 @@ until the condition is fixed.
 We encourage you to fix this by removing additional dividing buckets or bump the
 number of dividing buckets to 2.
 
-UNEVEN_WEIGHTS_STRETCH_MODE
+STRETCH_MODE_BUCKET_WEIGHT_IMBALANCE
 ___________________________
 
-The 2 dividing buckets must have equal weights when stretch mode is enabled.
-This warning suggests that the 2 dividing buckets have uneven weights after
-stretch mode is enabled. This is not immediately fatal, however, you can expect
-Ceph to be confused when trying to process transitions between dividing buckets.
+The 2 dividing buckets must have weights within a certain percentage difference 
+when stretch mode is enabled. This is determined by the configuration option
+``mon_stretch_max_bucket_weight_diff`` (default: 0.1).
 
-We encourage you to fix this by making the weights even on both dividing buckets.
+This is not immediately fatal, however, you can expect
+Ceph to have imbalance distribution and performance bottlenecks
```

Comment:
Ceph to experience performance bottlenecks and imbalanced data distribution

"imbalance" lacks the d. I changed the order so that adjective doesn't ambiguously refer to both distribution and bottlenecks.

Reply:
pushed the new change thanks!